src/pages/[platform]/reference/cli-commands/index.mdx

```diff
@@ -225,7 +225,7 @@ npx ampx generate outputs --stack amplify-nextamplifygen2-josef-sandbox-ca85e108
 
 #### Use with mobile applications
 
-Similar to `sandbox`, you can specify an alternate configuration file format by using `--format`:
+Similar to `sandbox`, you can specify an alternate outputs file format by using `--format`:
```

Comment:
This file only had this change from this PR: https://github.com/aws-amplify/docs/pull/7346/files#diff-61d6c0f99c1aa1d1c1e497cfcfc20a3ae905ccd606de326c7f416eeacf467977

The other lines were removed from `next-release/main`, but not `main`